apps/web/components/sections/home/join-us.tsx

```
@@ -0,0 +1,117 @@
+        <div className="grid grid-cols-1 md:grid-cols-3 gap-8 lg:gap-12 mb-16">
+          {features.map((feature, index) => (
+            <motion.div
+              key={feature.id}
+              initial={{ opacity: 0, y: 20 }}
+              whileInView={{ opacity: 1, y: 0 }}
+              viewport={{ once: true }}
+              transition={{ duration: 0.5, delay: index * 0.1 }}
+            >
+              <div
+                className="group h-full bg-white rounded-2xl p-8 shadow-sm hover:shadow-md transition-all duration-300"
+                role="article"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<article>


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)